### PR TITLE
Make serf  a module

### PR DESCRIPTION
**Pull request by kitschysynq**
Made serf a module so we could potentially build one cluster per region, or one per user, or one per datastructure, or one per...

### CODE REVIEW

**Comment by kitschysynq** [comment]
I didn't do anything about ssh keys. We'll need to think about how we want to track those for deployment purposes.

File: modules/serf/main.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
+variable "num_nodes" { }
+variable "region" { }
+
+variable "image" {
+	default = "ubuntu-x86_64"
```
Comment (cagedmantis) [inline modules/serf/main.tf]: Doesn't seem to be a recognized image.

Comment (kitschysynq) [inline modules/serf/main.tf]: You are correct. I forgot to fix during my merge. I'll update it with correct slug.

On January 31, 2016 12:12:44 PM EST, Carlos Amedee notifications@github.com wrote:

> > @@ -0,0 +1,74 @@
> > +variable "num_nodes" { }
> > +variable "region" { }
> > +
> > +variable "image" {
> > -    default = "ubuntu-x86_64"
> 
> Doesn't seem to be a recognized image. 
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/criprooff/infrastructure/pull/3/files#r51363759

## 

/thor

**Comment by cagedmantis** [comment]
Otherwise :+1:

**Comment by kitschysynq** [comment]
That should have it

**Pull request merged**